Extensions/Dice.json

```diff
@@ -0,0 +1,1587 @@
+{
+  "author": "",
+  "description": "### What it helps you achieve :-\n\n> As the name might suggest it's an extension that makes it easy to have dice in your game.\n> 1) You can roll a dice manually.\n> 2) Turn on a parameter so that if your dice moves above a speed limit it will roll automatically (you can specify the limit).\n> 3) Turn this other parameter on which does the same thing but checks if the dice is spinning (you can specify the limit).\n> 4) You can have any number of sides to the dice and have anything on those sides doesn't have to be numbers or dots\n> 5) You can add an animation for when it rolls and even customize how long should the dice roll for (i.e. how long should that animation play for)\n\n### How to set it up:-\n\n> Step 1 - Create a sprite object and add all of the individual sides of the dice as individual animations (please add the dice roll animation after or before all of the sides and not in the middle)\n\n> Step 2 - Each of the animations representing a side should have a number as their name this would represent the value of that side\n\n> Step 3 - Add the dice behavior to the object and fill the parameters according to your need\n\n> Step 4 - Enjoy\n\n### How to use the features:- \n\n> 1) You can roll the dice manually by using the \"Roll\" action under the dice group\n\n> 2) You can extract your dice roll values using the \"Dice Roll Value\" expression\n\n> 3) You can use the \"Edit Properties\" action to change the properties of dice through an event\n\n",
```

Comment:
I recommend you to use [this website](https://stackedit.io/app#) to get a preview.
Please:
- Remove one `#` on title lines
- Remove all `>`
- Replace the symbol `:-` by a simple `:`
- Use the list syntax instead of `Step 1 2 3...`